redbot/cogs/mutes/mutes.py

```diff
@@ -0,0 +1,1212 @@
+            except Exception:
+                log.error("error checking channel unmutes", exc_info=True)
+            await asyncio.sleep(30)
+
+    async def _clean_tasks(self):
+        log.debug("Cleaning unmute tasks")
+        is_debug = log.getEffectiveLevel() <= logging.DEBUG
+        for task_id in list(self._unmute_tasks.keys()):
+            task = self._unmute_tasks[task_id]
+
+            if task.cancelled():
+                self._unmute_tasks.pop(task_id, None)
+                continue
+
+            if task.done():
+                try:
+                    r = task.result()
+                except Exception:
+                    if is_debug:
+                        log.exception("Dead task when trying to unmute")
+                self._unmute_tasks.pop(task_id, None)
+
+    async def _handle_server_unmutes(self):
+        log.debug("Checking server unmutes")
+        to_clear = []
+        for g_id, mutes in self._server_mutes.items():
+            to_remove = []
+            guild = self.bot.get_guild(g_id)
+            if guild is None:
+                continue
+            for u_id, data in mutes.items():
+                if data["until"] is None:
+                    continue
+                time_to_unmute = data["until"] - datetime.now(timezone.utc).timestamp()
+                if time_to_unmute < 60:
+                    self._unmute_tasks[f"{g_id}{u_id}"] = asyncio.create_task(
+                        self._auto_unmute_user(guild, data)
+                    )
+                    to_remove.append(u_id)
+            for u_id in to_remove:
+                del self._server_mutes[g_id][u_id]
+            if self._server_mutes[g_id] == {}:
+                to_clear.append(g_id)
+            await self.config.guild(guild).muted_users.set(self._server_mutes[g_id])
+        for g_id in to_clear:
+            await self.config.guild_from_id(g_id).muted_users.clear()
+
+    async def _auto_unmute_user(self, guild: discord.Guild, data: dict):
+        delay = data["until"] - datetime.now(timezone.utc).timestamp()
+        if delay < 1:
+            delay = 0
+        await asyncio.sleep(delay)
+        try:
+            member = guild.get_member(data["member"])
+            author = guild.get_member(data["author"])
+            if not member:
+                return
+            success, message = await self.unmute_user(guild, author, member, _("Automatic unmute"))
+            if success:
+                try:
+                    await modlog.create_case(
+                        self.bot,
+                        guild,
+                        datetime.now(timezone.utc),
+                        "sunmute",
+                        member,
+                        author,
+                        _("Automatic unmute"),
+                        until=None,
+                    )
+                except RuntimeError as e:
+                    log.error(_("Error creating modlog case"), exc_info=e)
```

Comment:
Logging should not be translated.
```suggestion
                    log.error("Error creating modlog case", exc_info=e)
```